src/Stripe.net/Services/Subscriptions/StripeSubscriptionCreateOptions.cs

```diff
@@ -37,6 +37,9 @@ public class StripeSubscriptionCreateOptions : SubscriptionSharedOptions
         [JsonProperty("subscription_items_array")]
         public List<StripeSubscriptionItemOption> Items { get; set; }
 
+        [JsonProperty("tax_percent")]
```

Comment:
The `StripeSubscriptionCreateOptions` class inherits from `SubscriptionSharedOptions` which already defines `TaxPercent` as you can see here: https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/Subscriptions/SubscriptionSharedOptions.cs#L63